Src/PerformanceCollector/Shared/Implementation/IPerformanceCollector.cs

```diff
@@ -25,12 +25,12 @@ internal interface IPerformanceCollector
         /// <summary>
         /// Registers a counter using the counter name and reportAs value to the total list of counters.
         /// </summary>
-        /// <param name="perfCounterName">Name of the performance counter.</param>
+        /// <param name="perfCounter">Name of the performance counter.</param>
```

Comment:
why rename? Is it not the name any more? If not the name - change the description of the argument.